.run/Run process-agent.run.xml

```diff
@@ -15,7 +15,7 @@
       <env name="SCRIPT_TO_RUN" value=".run/run.sh" />
       <env name="BINARY_TO_RUN" value="./bin/process-agent/process-agent" />
       <env name="DLV_PORT_TO_BIND" value="2347" />
-      <env name="BINARY_ARGUMENTS" value="&quot;--cfgpath ./bin/agent/dist/datadog.yaml&quot;" />
+      <env name="BINARY_ARGUMENTS" value="--cfgpath ./dev/dist/datadog.yaml" />
```

Comment:
change the path in the other scripts as well

Reply:
Fixed